src/api/operator/utils.ts

```diff
@@ -49,3 +48,100 @@ export function getOrderStatus(order: RawOrder): OrderStatus {
     return 'open'
   }
 }
+
+/**
+ * Get order filled amount, both as raw amount (in atoms) and as percentage (from 0 to 1)
+ *
+ * @param order The order
+ */
+export function getOrderFilledAmount(order: RawOrder): { amount: BigNumber; percentage: BigNumber } {
+  let executedAmount, totalAmount
+
+  if (order.kind === 'buy') {
+    executedAmount = new BigNumber(order.executedBuyAmount)
+    totalAmount = new BigNumber(order.buyAmount)
+  } else {
+    executedAmount = new BigNumber(order.executedSellAmount).minus(order.executedFeeAmount)
```

Comment:
since you're subtracting it here it only seems logical but maybe I'm missing sth

Reply:
I don't get the question :confused: